STATUS: INLINE SUCCESSFUL
MODIFIED: io.github.dddplus.runtime.registry.IntegrationTest.testPolicy
MODIFIED FILE: /home/ecelrc/students/cli7/tool/automated_script/funkygao/cp-ddd-framework/dddplus-test/src/test/java/io/github/dddplus/runtime/registry/IntegrationTest.java
CLEANER: io.github.dddplus.runtime.registry.IntegrationTest.stepsExecTemplateWithRollback
CLEANER FILE: /home/ecelrc/students/cli7/tool/automated_script/funkygao/cp-ddd-framework/dddplus-test/src/test/java/io/github/dddplus/runtime/registry/IntegrationTest.java
POLLUTER: io.github.dddplus.runtime.registry.BadOnPurposeTest.specificationNotAnnotated
POLLUTER FILE: /home/ecelrc/students/cli7/tool/automated_script/funkygao/cp-ddd-framework/dddplus-test/src/test/java/io/github/dddplus/runtime/registry/BadOnPurposeTest.java
ORIGINAL CLEANER SIZE: 9
NEW CLEANER SIZE: 4
ELAPSED TIME: 1551.4739999771118
==========================
@@ -204,0 +204,6 @@
+ {
+     fooModel.setB2c(false);
+     fooModel.setWillRollback(true);
+     List<String> steps = DDD.findAbility(DecideStepsAbility.class).decideSteps(fooModel, Steps.Submit.Activity);
+     submitStepsExec.execute(Steps.Submit.Activity, steps, fooModel);
+ }
